api/spec/v1.yaml

```diff
@@ -1874,8 +1874,14 @@ components:
           description: The public key identifying this Validator's node.
           example: *node_id_1
         escrow:
+          allOf: [$ref: '#/components/schemas/Escrow']
+          description: The escrow account data for this validator.
+        voting_power:
           allOf: [$ref: '#/components/schemas/TextBigInt']
-          description: The amount staked.
+          description: The voting power of this Validator.
+        voting_power_total:
```

Comment:
this (and `active_balance_total` below) seems a bit out of place here; but i'm not sure whether it's worth creating a separate endpoint to return this network-level info.

Reply:
ya I don't know what to do about this

Reply:
oh clarification on this and active_balance_total below: "active" refers to all validators, i.e. in the current committee? e.g. if a validator is down mid-epoch, it's still counted in these right?

Reply:
> "active" refers to all validators, i.e. in the current committee? e.g. if a validator is down mid-epoch, it's still counted in these right?

Yep

Reply:
> but i'm not sure whether it's worth creating a separate endpoint to return this network-level info

Instead of introducing a new endpoint we could also update `consensus/validators` response type which would include `NetworkInfo` next to `ValidatorList`. 

But it is probably also fine if we keep it as is, if that works better for frontend.